siemens/run_siemens.sh

```diff
@@ -17,7 +17,7 @@ EOF
 )
 fail_cmd=$(cat <<EOF
        grep -E --color \
-           "(^\*\* Fatal[ :]|^\*\* Error[ :]|^\*\* [^W]|UVM_FATAL[ :]@)" \
+           "(^\*\* Fatal[ :]|^\*\* Error[   :]|^\*\* [^W]|UVM_FATAL[ :]@)" \
```

Comment:
Is this extra character deliberate? Is there something in autocheck logs that need it?

Reply:
I might have be misunderstanding regex.  Some Error words have multiple spaces after it but I think `[ :]` says match a ` ` space or a `:` right?  So I should reverse this change right?

Reply:
Yes, that is not a usefully change. It says "match a space or a space or a colon after the word `Error`.